adr/0009-frontend-directory-structure-and-architecture.md

```diff
@@ -0,0 +1,64 @@
+# 9. Frontend directory structure and architecture
+
+Date: 2023-05-16
+
+## Status
+
+Accepted
+
+## Context
+
+Our software project has a frontend that needs to be organized to facilitate maintenance, scalability, modularity, and ease of navigation. The current directory structure is outdated and hard to manage. As software engineers, we propose a new directory structure based on some criteria.
+
+## Decision
+
+We propose the following new directory structure for the frontend of our software project:
```

Comment:
Could we add some link to the react docs?
https://redux.js.org/faq/code-structure/

Reply:
Doin' this right now :D